adam-core/src/main/scala/org/bdgenomics/adam/rdd/variation/VariantContextRDD.scala

```diff
@@ -60,20 +60,19 @@ case class VariantContextRDD(rdd: RDD[VariantContext],
    * @param ann Annotation RDD to join against.
    * @return Returns a VariantContextRDD where annotations have been filled in.
    */
-  def joinDatabaseVariantAnnotation(ann: DatabaseVariantAnnotationRDD): VariantContextRDD = {
+  def joinVariantAnnotations(ann: VariantAnnotationRDD): VariantContextRDD = {
     replaceRdd(rdd.keyBy(_.variant)
```

Comment:
We might want to open a ticket for this, but after #1216 this should probably be implemented using a region join instead of a Spark core `leftOuterJoin`.